### PR TITLE
[PDMO 90]: Add plus icon for addresses

### PR DESCRIPTION
**Pull request by brayoh**
## JIRA TICKET NAME:
[[PDMO-90]: Add plus icon for addresses](https://issues.openmrs.org/browse/PDMO-90)

## SUMMARY:
The add icon doesn't show in the addresses section, this should be looked into and fixed.

### CODE REVIEW

**Comment by Farhanisto** [comment]
Looks good

**Review by malmike — COMMENTED** [review]
Looks Good

File: app/css/obsadmin.css [hunk 1]
```diff
@@ -41,30 +42,30 @@ header .logo img {
...
-    color: white;
-    margin-right: 10px;
-    font-size:20px;
+  color: white;
+  margin-right: 10px;
+  font-size: 20px;
 }
 
 .fa-user:before, .fa-map-marker:before {
-    margin-right: 10px;
```
Comment (dkayiwa) [inline app/css/obsadmin.css]: Are all changes in this file and others simply about "Add plus icon for addresses"?

Comment (brayoh) [reply app/css/obsadmin.css]: not really, I changed the indentation and removed some unused code, just some bit of clean up.

Comment (dkayiwa) [reply app/css/obsadmin.css]: That should be done in a separate ticket and pull request. This one should have only the bare minimum required by the ticket.

Comment (brayoh) [reply app/css/obsadmin.css]: The only added feature is the plus icon for addresses, the rest is just code clean up.

Comment (brayoh) [reply app/css/obsadmin.css]: it's just indentation fixes not a feature

Comment (brayoh) [reply app/css/obsadmin.css]: @dkayiwa you mean  a ticket for indentation?

Comment (dkayiwa) [reply app/css/obsadmin.css]: yes

**Comment by brayoh** [comment]
@dkayiwa  done

File: app/js/components/subAddress/addressForm.jsx [hunk 2]
```diff
@@ -325,7 +325,7 @@ export default class AddressForm extends React.Component {
             </Modal>
           :
           <div>
-            {this.state.secondryAction === 'new' ?
+            {this.props.address.secondryAction === 'new' ?
```
Comment (dkayiwa) [inline app/js/components/subAddress/addressForm.jsx:328]: Is this the only change required to Add plus icon for addresses?

Comment (brayoh) [reply app/js/components/subAddress/addressForm.jsx:328]: Yes it is.

Comment (dkayiwa) [reply app/js/components/subAddress/addressForm.jsx:328]: So do you now see how easier it is to review the actual change in comparison to when you had all the other formatting changes?

Comment (brayoh) [reply app/js/components/subAddress/addressForm.jsx:328]: Yes it is, sorry about that.

Comment (dkayiwa) [reply app/js/components/subAddress/addressForm.jsx:328]: And by the way, formatting and cleanup is a very good thing. It is just that it should not mixed with other changes. 😊

**Pull request merged**